Integrations/PaloAltoNetworks_Traps/PaloAltoNetworks_Traps.yml

```diff
@@ -0,0 +1,377 @@
+category: Authentication
+commonfields:
+  id: Traps
+  version: -1
+configuration:
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Server URL
+  name: url
+  required: true
+  type: 0
+- display: Application ID
+  name: application_id
+  required: true
+  type: 0
+- display: Private Key
+  name: private_key
+  required: false
```

Comment:
are you sure `private_key` should not required?

Reply:
- [x] Fixed